Orchestrator.Services/ServiceBusService.cs

```diff
@@ -11,18 +11,18 @@ namespace Orchestrator.Services
 {
     public class ServiceBusService : IServiceBusService
     {
-        private readonly IQueueClient _queueClient;
+        private readonly ServiceBusClient _serviceBusClient;
```

Comment:
Does this need to be a field here? Since we currently just need the sender

Reply:
This is a hint to the devs to NOT dispose the object: 
![image](https://user-images.githubusercontent.com/67424843/236221526-c17c55ea-cce6-48cb-a769-4fa5bdb58af0.png)
https://learn.microsoft.com/en-us/dotnet/api/overview/azure/messaging.servicebus-readme?source=recommendations&view=azure-dotnet#client-lifetime
If it's disposed, the ServiceBusSender will also be disposed.

Reply:
![image](https://user-images.githubusercontent.com/67424843/236227161-9a522ce6-1799-4db9-9590-7efc5204f2cd.png)
Better?